api/registrations/views.py

```diff
@@ -559,6 +559,8 @@ class RegistrationFilesList(NodeFilesList, RegistrationMixin):
     """
     view_category = 'registrations'
     view_name = 'registration-files'
+
+    ordering_fields = ['modified', 'name']
```

Comment:
@adlius could you make this a tuple to be more consistent with the other ordering fields? Also tuples are immutable so a little bit of a better data type for ordering things by declaration.

Reply:
DRF document examples uses lists. And IIRC tuples doesn't work for `ordering_fields` when I tried it locally.